pkg/configuration/parser.go

```diff
@@ -119,7 +119,7 @@ func (p *nsxConfigParser) removeVMsWithoutGroups() {
 	toRemove := []topology.Endpoint{}
 	for vm, groups := range p.configRes.GroupsPerVM {
 		if len(groups) == 0 {
-			logging.Debugf("ignoring VM without groups: %s", vm.Name())
+			logging.Warnf("ignoring VM without groups: %s. address %s", vm.Name(), vm.IPAddressesStr())
```

Comment:
- change `Warnf` to `Debugf` (all warnings are currently issued through debug log.. will open a separate issue to classify which debug warnings should be changed. the current run without verbose log should be clean of log warnings, achieved by debug level warnings only). 

- avoid reporting "address" suffix if it is empty:
`2025/03/30 14:13:16 WARN        ignoring VM without groups: test-2. address`

Reply:
fixed